File: 412. Fizz Buzz/fizzBuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        #Make empty list
        output = []
        #Iterate over each num in range 1, n+1
        #To speed up run time, start off saying if num%15 instead of %3 and %5.
        for num in range(1, n+1):
            if num%15 == 0:
                output.append("FizzBuzz")
            elif num%3 == 0:
                output.append("Fizz")
            elif num%5 == 0:
                output.append("Buzz")
            #Important: appending str(num) in one step speeds things up
            else:
                output.append(str(num))
        return output 

File: 412. Fizz Buzz/fizzBuzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        //Create empty String ArrayList
        List<String> myList = new ArrayList<>();
        //Iterate with i, as long as i is less than n. Use 15 to speed up code
        for (int i = 1; i<=n; i++){
            if (i%15 == 0){
                myList.add("FizzBuzz");
            }
            else if (i%3 == 0){
                myList.add("Fizz");
            }
            else if (i%5 == 0){
                myList.add("Buzz");
            }
            else{
                myList.add(String.valueOf(i));
            }
        }
        return myList;
    }
}
